### PR TITLE
[IMP] open_academy: Add session menu and changes to the course menu

### PR DESCRIPTION
**Pull request by rolandojduartem**
**Course Menu**
In this pull request, I added the tab and search view to the course menu.
Screenshot from 2022-01-10 19-47-12

**Session Menu**
I created this session menu with its model, demo data and the tree, form and search views.
Screenshot from 2022-01-10 19-45-57

Screenshot from 2022-01-10 19-46-40

### CODE REVIEW

**Comment by rolandojduartem** [comment]
@payen000 could you review this pull request, please?

File: open_academy/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
 id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
-access_open_academy_model_course,access_open_academy_model_course,model_course,base.group_user,1,1,1,1
\ No newline at end of file
+access_open_academy_model_course,access_open_academy_model_course,model_course,base.group_user,1,1,1,1
+access_open_academy_model_session,access_open_academy_model_session,model_session,base.group_user,1,1,1,1
```
Comment (payen000) [inline open_academy/security/ir.model.access.csv:3]: Please add newline at the end of file

**Review by payen000 — CHANGES_REQUESTED** [review]
Please address this comment :)

File: open_academy/security/ir.model.access.csv (hunk 1 above)
Comment (rolandojduartem) [reply open_academy/security/ir.model.access.csv]: It is resolved!

**Review by payen000 — APPROVED** [review]
+1 looks good! :)

File: open_academy/views/session.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <field name="start_date"/>
+            <field name="duration" widget='timesheet_uom'/>
+            <field name="number_seat"/>
+          </search>
+        </field>
+      </record>
+      <record id='session_action_display_view' model='ir.actions.act_window'>
+        <field name='name'>Session</field>
+        <field name='res_model'>session</field>
+        <field name='view_mode'>tree,form,search</field>
```
Comment (Jerudiel) [inline open_academy/views/session.xml:54]: Please address this comment c:

Is it necessary add "search" to this field?

Comment (rolandojduartem) [reply open_academy/views/session.xml:54]: When I did not add it, the app did not work for me. So, I added and everything worked fine.

Comment (Jerudiel) [reply open_academy/views/session.xml:54]: Ok, it was just a doubt.

**Comment by Jerudiel** [comment]
+1 looks good!

File: open_academy/views/session.xml (hunk 2 above)
Comment (payen000) [reply open_academy/views/session.xml:54]: Sorry @Jerudiel, I still don't know enough to answer your question, I'm a little behind in the training document, but Francisco surely could answer that in a few minutes! :D

**Comment by rolandojduartem** [comment]
@frahikLV , I got +1 from teammates, could you review this PR?

File: open_academy/views/course.xml [hunk 3]
```diff
@@ -9,9 +9,11 @@
             <group>
               <field name="title"/>
             </group>
-            <group>
-              <field name="description"/>
-            </group>
+            <notebook>
+              <sheet>
```
Comment (frahikLV) [inline open_academy/views/course.xml:13]: change sheet tag to page tag, sheet it's to draw the following spaces (in red)
image

Please change to
```suggestion
              <page>
```

File: open_academy/views/course.xml [hunk 4]
```diff
@@ -26,10 +28,20 @@
...
+        </search>
+      </field>
+    </record>
     <record id='course_action_display_view' model='ir.actions.act_window'>
       <field name='name'>Course</field>
       <field name='res_model'>course</field>
-      <field name='view_mode'>tree,form</field>
+      <field name='view_mode'>tree,form,search</field>
     </record>
   </data>
```
Comment (frahikLV) [inline open_academy/views/course.xml:46]: <data> tag it's not being used, please remove it

File: open_academy/views/session.xml [hunk 5]
```diff
@@ -0,0 +1,57 @@
+<odoo>
+    <data>
```
Comment (frahikLV) [inline open_academy/views/session.xml:2]: <data> tag it's not being used, please remove it

File: open_academy/views/session.xml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+              <group>
+                <field name="name"/>
+              </group>
+              <notebook>
+                <page string="Start date">
+                  <field name="start_date"/>
+                </page>
+                <page string="Duration">
+                    <field name="duration" widget='timesheet_uom'/>
+                  </page>
```
Comment (frahikLV) [inline open_academy/views/session.xml:18]: Please unindent this line

File: open_academy/views/session.xml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                  <page string="Number of seats">
```
Comment (frahikLV) [inline open_academy/views/session.xml:19]: Please unindent this line

File: open_academy/views/session.xml [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+                    <field name="number_seat"/>
```
Comment (frahikLV) [inline open_academy/views/session.xml:20]: Please unindent this line

File: open_academy/views/session.xml [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+                  </page>
```
Comment (frahikLV) [inline open_academy/views/session.xml:21]: Please unindent this line

File: open_academy/views/session.xml [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+            <field name="number_seat"/>
+          </search>
+        </field>
+      </record>
+      <record id='session_action_display_view' model='ir.actions.act_window'>
+        <field name='name'>Session</field>
+        <field name='res_model'>session</field>
+        <field name='view_mode'>tree,form,search</field>
+      </record>
+    </data>
```
Comment (frahikLV) [inline open_academy/views/session.xml:56]: remove this tag

File: open_academy/views/session.xml (hunk 2 above)
Comment (frahikLV) [reply open_academy/views/session.xml:54]: @rolandojduartem remove search from view_mode

**Review by frahikLV — CHANGES_REQUESTED** [review]
@rolandojduartem please address these comments

**Comment by rolandojduartem** [comment]
@frahikLV changes were applied, could you review it?

**Comment by frahikLV** [comment]
LGMT! @rolandojduartem I'm going to merge this PR

**Pull request merged**